backend/sirius-web-view/src/main/java/org/eclipse/sirius/web/view/NodeStyle.java

```diff
@@ -240,4 +242,27 @@
      * @generated
      */
     void setStrikeThrough(boolean value);
+
+    /**
+     * Returns the value of the '<em><b>Size Computation Expression</b></em>' attribute. The default value is
+     * <code>"1"</code>. <!-- begin-user-doc --> <!-- end-user-doc -->
+     *
+     * @return the value of the '<em>Size Computation Expression</em>' attribute.
+     * @see #setSizeComputationExpression(String)
+     * @see org.eclipse.sirius.web.view.ViewPackage#getNodeStyle_SizeComputationExpression()
+     * @model default="1"
+     * @generated
+     */
+    String getSizeComputationExpression();
```

Comment:
I wonder why you have a different name in the metamodel and in the user interface. Is `sizeComputationExpression` the Sirius RCP name?

Reply:
`sizeComputationExpression` is indeed the name in Sirius Desktop's metamodel.
Seeing it in the UI, "Computation Expression" felt redundant, and given that the View DSL is not tied to Sirius Desktop I simplified it.
I should have simplified it directly in the metamodel. Too late for this PR but I'll do that in a future one.